docs/guides/audio-config.md

```diff
@@ -41,7 +41,6 @@ An example to get PipeWire working on Ubuntu, which uses PulseAudio by default i
 # Issues
 
 - Some people are unable to get audio input to work. You may have to use a separate microphone.
-- All of apple's fancy tuning of the speakers is done in macOS, we don't have anything like that at the moment.
```

Comment:
All of apple's fancy tuning of the speakers is done in macOS, similar configuration is currently available only for MacBook Pro 16 inch 2019 only.

This sounds better to me.

Reply:
ok

Reply:
I think this is clearer.

All of apple's fancy tuning of the speakers is done in macOS, but a similar configuration is currently available for only the MacBook Pro 16 inch 2019.

Reply:
This also looks good

Reply:
I'm also glad to see this line replaced!
I have a 16,1 and will try this out eventually, but to be honest I was taken a little aback with the complexity of what it takes to make this work.

But maybe it will inspire others with different models to at least investigate and contribute.

Reply:
this is just a initial step to notify users about the new settings. 
there will be more revision to make the installation more simple